model.py

```diff
@@ -1,19 +1,56 @@
+from dao import Dao
+
 class Model:
+    data_access = Dao()
 
-    def __init__(self, name, function):
+    def __init__(self, name, function, _id):
         self.name = name
         self.function = function
+        self._id = _id
+    @classmethod
+    def update(cls, name, function):
+        thing = cls.data_access.delete(name)
```

Comment:
why are you deleting in order to update ? what if they only want to update a single value ? this will give the item an entirely new ID .. we're not creating another record, we're updating a single record

Reply:
changed